### PR TITLE
feat: auth support

### PR DESCRIPTION
**Pull request by cl-shaharshavit-rnd**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔/✖                                                                        |
| New feature     | ✔/✖                                                                        |
| Breaking change | ✔/✖                                                                        |
| Deprecations    | ✔/✖                                                                        |
| Documentation   | ✔/✖                                                                        |
| Tests added     | ✔/✖                                                                        |
| Chore            | ✔/✖                                                                       |

Related issues: #XXX , #XXX ...
Closes #XXX ...

Further  information:

### CODE REVIEW

**Review by CL-SHLOMIKONCHA — APPROVED** [review]

File: helm/config/envoy.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                      with_request_body:
+                        max_request_bytes: {{ .Values.envoy.maxRequestBytes }}
+                        allow_partial_message: true
+                      failure_mode_allow: false
+                      grpc_service:
+                        google_grpc:
+                          target_uri: localhost:{{ .Values.opa.port }}
+                          stat_prefix: ext_authz
+                        timeout: 0.5s
+                      metadata_context_namespaces:
```
Comment (asafMasa) [inline helm/config/envoy.yaml:45]: check what relevant

File: helm/values.yaml [hunk 2]
```diff
@@ -27,12 +31,42 @@ cloudProvider:
 
 image:
   repository: exporter-trigger
-  tag: 'latest'
+  tag: 'v2.2.0'
```
Comment (asafMasa) [inline helm/values.yaml]: latest

File: helm/values.yaml [hunk 3]
```diff
@@ -27,12 +31,42 @@ cloudProvider:
...
+
+opa:
+  image: openpolicyagent/opa
+  imageTag: 0.29.4-envoy-3
+  port: 9191
+  enableLogging: false
+  resources:
+    enabled: true
```
Comment (asafMasa) [inline helm/values.yaml:42]: resources isn't config?

File: helm/values.yaml [hunk 4]
```diff
@@ -27,12 +31,42 @@ cloudProvider:
...
+        cpu: 100m
+        memory: 128Mi
+envoy:
+  image: envoyproxy/envoy
+  imageTag: v1.20.7
+  port: 8000
+  maxRequestBytes: 8192
+  requestedDomain: rasterExport
+  resources:
+    enabled: true
```
Comment (asafMasa) [inline helm/values.yaml:57]: resources isn't config?

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍

File: helm/values.yaml (hunk 3 above)
Comment (cl-shaharshavit-rnd) [reply helm/values.yaml:42]: this is the config!

**Review by asafMasa — APPROVED** [review]

**Pull request merged**